### DISCUSSION TITLE
Pflügen im Beete Modus

### DISCUSSION THREAD

**Discussion by lukas30t** [Courseplay/Courseplay_FS22#3167] [Ideas]
Hallo zusammen,

Vielleicht mache ich da etwas falsch, aber wenn ich mit CP Beete Modus für pflügen benutze, fahren die Trecker nicht von Ihnen nach außen. Bei den Dreschern funktioniert das einwandfrei. 

Ich benutze nicht drehbare Pflüge. Ich finde, das im diesem Modus, die Trecker so wie die Drescher fahren sollten. Das wurde dann viel realistischer aussehen. 

Vielleicht könnte man einen extra Modus für nicht drehbare Pflüge implementieren, wo die Trecker sich genauso wie die Drescher verhalten?

MfG
Lukas

**Comment by Tensuko** [comment]
Beete ist auch nicht fürs Pflügen gemacht.
Benutze hoch/runter dafür. Beete bringt keinen Vorteil für andere Feldarbeiten!
